docs/cicd_tools/common.md

```diff
@@ -0,0 +1,69 @@
+# Common module
+
+This module exposes helper functions that may be used (and shared) across all modules, or that don't
+serve a specific module's purpose to fit within it.
+
+## Definition
+
+The module ID is `common`
+
+All functions exposed by this module will use the namespaced prefix:
+
+```
+cicd::common::
+```
+
+## Usage
+
+Use the `common` id to load the module
+
+```
+CICD_TOOLS_URL="https://raw.githubusercontent.com/RedHatInsights/cicd-tools/main/src/bootstrap.sh"
+# shellcheck source=/dev/null
+source <(curl -sSL "$CICD_TOOLS_URL") common
+```
+
+This should load the function:
```

Comment:
I don't understand the flow here. We're talking about the `cicd::common::` module but then you show an example of loading a `cicd::container:: function` is this a mistake? If not then I'm not super clear on what the flow is here.

Reply:
I'm so sorry, I changed the structure of the README's of each module so they all shared the same structure and I forgot to update the `common` one, I've pushed it now.